packages/next/src/server/lib/router-utils/setup-dev-bundler.ts

```diff
@@ -21,6 +21,7 @@ import findUp from 'next/dist/compiled/find-up'
 import { buildCustomRoute } from './filesystem'
 import * as Log from '../../../build/output/log'
 import HotReloaderWebpack from '../../dev/hot-reloader-webpack'
+import { getVersionInfo } from '../../dev/hot-reloader-webpack'
```

Comment:
Wondering if we should combine this import with the one above? Not sure if there is standard or style that says we should do it this way or the other.